x-pack/test/fleet_api_integration/apis/preconfiguration/preconfiguration.ts

```diff
@@ -42,11 +42,7 @@ export default function (providerContext: FtrProviderContext) {
         .send({})
         .expect(200);
 
-      expect(body).to.eql({
-        packages: [],
-        policies: [],
-        nonFatalErrors: [],
-      });
+      expect(body.nonFatalErrors).to.eql([]);
```

Comment:
This assertion was failing because we actually have some values in `packages` now. The essence of this test is to assert that no errors are thrown, so this feels like a more explicit assertion anyway.